examples/all-clusters-app/all-clusters-common/gen/callback.h

```diff
@@ -1936,12 +1936,6 @@ bool emberAfBarrierControlClusterBarrierControlGoToPercentCallback(chip::app::Co
 
 bool emberAfBarrierControlClusterBarrierControlStopCallback(chip::app::Command * commandObj);
 
-/**
- * @brief Basic Cluster MfgSpecificPing Command callback
- */
-
-bool emberAfBasicClusterMfgSpecificPingCallback(chip::app::Command * commandObj);
```

Comment:
Please do not remove that.

Reply:
Do we have special reason not to remove it? AFAIS, the spec does not include `MfgSpecific` flag for this.

Reply:
> Do we have some reason to keep it? AFAIS, the spec does not include `MfgSpecific` flag for this.

It is currently use into the iOS app. My understanding is that the spec related bits for this kind of things are located into the `Manufacturer Specific Extensions` parts ?

Reply:
I see, however seems CHIP is using a prefix in identifiers instead of having a manufacturer id field in the protocol.

I will temporary remove the mfg code field in XML, and see how to deal with it later.

Reply:
Just read the spec, the MFG code becomes a prefix in the identifiers, in this case, we need to update the ZAP and related typedef in CHIP, while IM can move forward and no special changes are required.